Dockerfile

```diff
@@ -14,12 +14,23 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         git \
         locales \
         unzip \
+        nodejs \
```

Comment:
Dost dlho som riesil ako to implementovat ... zistil som, ze budem potrebovat https://github.com/keboola/php-csvmap, ... to je kniznica, ktora podla zadanych pravidiel namapuje JSON na (viacero) CSV, .... kedze CosmoDB vracia JSON, potobne ako MongoDB ... a kedze budeme potrebovat tuto `csvmap` kniznicu, ktora je v PHP (a prepisat ju do JS by trvalo velmi dlho) .... tak samotna komponenta je napisana v PHP, a vola NodeJS process.

Ako tu bude fungovat?

- Pre NodeJs existuje oficialny driver, ktory pre PHP nemame (ani ziadny co by vyzeral pouzitelne).
- Kod v NodeJS sa pripoji k databaze a na vystup STDOUT bude vypisovat JSON dokumenty, medzi ktorymi bude oddelovac `\n---\n`
- PHP bude asynchronne (teda nie az po skonceni procesu) citat STDOUT procesu a parsovat JSON na pole
- Pole bude v PHP ulozene do CSV pomocou kniznice `csvmap`
- Kedze sa to riesi asynchronne a vsetky vysledky sa neukladaju na disk alebo do pamate, tak to bude rychlejsie a bez problemov.
- Nakoniec sa vygeneruje manifest.